tests/test_component/test_amgr.py

```diff
@@ -72,6 +71,22 @@ def test_amgr_read_config(self, mocked_init, d):
                               rts_config=None,
                               base_path=None)
 
+        # test configuring services (service tasks)
+
+        with self.assertRaises(ree.EnTKTypeError):
+            # only re.Task objects are acceptable
+            amgr.services = [None]
+
+        services = [re.Task()]
+        amgr.services = services
+        # without initialized ResourceManager
+        self.assertEqual(amgr._services, services)
+
+        amgr._rmgr = mock.Mock()
+        amgr.services = services
+        # with initialized ResourceManager
+        self.assertEqual(amgr._rmgr.services, services)
+
```

Comment:
The third option would be to attach services to the amgr, then attach resources to the amgr - then the rmgr should know the services also.

Reply:
done